libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -179,39 +190,17 @@ class MapboxTripNotification constructor(
         }
     }
 
-    private fun generateArrivalTime(
-        routeProgress: RouteProgress,
-        time: Calendar
-    ): String? =
-        ifNonNull(routeProgress.currentLegProgress()) { currentLegProgress ->
-            val legDurationRemaining = currentLegProgress.durationRemaining()
-            val timeFormatType = navigationOptions.timeFormatType()
-            val arrivalTime = formatTime(
-                time,
-                legDurationRemaining.toDouble(),
-                timeFormatType,
-                DateFormat.is24HourFormat(applicationContext)
-            )
-            String.format(etaFormat, arrivalTime)
-        }
-
     private fun updateNotificationViews(routeProgress: RouteProgress) {
-        buildRemoteViews()
         updateInstructionText(routeProgress.bannerInstructions())
         updateDistanceText(routeProgress)
         generateArrivalTime(routeProgress, Calendar.getInstance())?.let { formattedTime ->
             updateViewsWithArrival(formattedTime)
-            routeProgress.currentLegProgress()?.upcomingStep()?.let { step ->
-                updateManeuverImage(step)
-            } ?: routeProgress.currentLegProgress()?.currentStepProgress()?.step()
+            val step = routeProgress.currentLegProgress()?.upcomingStep()
+                ?: routeProgress.currentLegProgress()?.currentStepProgress()?.step()
+            step?.let { updateManeuverImage(it) }
```

Comment:
The maneuver image is only supposed to show the upcoming maneuver. With this change it will show the current manuever if upcoming is null. That might be misleading. I don't think this change is necessary.

Reply:
I was embarassed by
`?: routeProgress.currentLegProgress()?.currentStepProgress()?.step()` 
line. 
If we're going to use maneuver image only for upcoming maneuver it's completely useless line

Reply:
@abhishek1508 Does it look good now?